Doc/library/idle.rst

```diff
@@ -660,29 +660,58 @@ functions to be used from IDLE's Python shell.
 Command line usage
 ^^^^^^^^^^^^^^^^^^
 
-.. code-block:: none
+.. program:: idle
 
-   idle.py [-c command] [-d] [-e] [-h] [-i] [-r file] [-s] [-t title] [-] [arg] ...
+IDLE can be invoked from the command line with various options. The general syntax is:
 
-   -c command  run command in the shell window
-   -d          enable debugger and open shell window
-   -e          open editor window
-   -h          print help message with legal combinations and exit
-   -i          open shell window
-   -r file     run file in shell window
-   -s          run $IDLESTARTUP or $PYTHONSTARTUP first, in shell window
-   -t title    set title of shell window
-   -           run stdin in shell (- must be last option before args)
+.. code-block:: bash
 
-If there are arguments:
+   python -m idlelib [options] [file ...]
 
-* If ``-``, ``-c``, or ``r`` is used, all arguments are placed in
-  ``sys.argv[1:...]`` and ``sys.argv[0]`` is set to ``''``, ``'-c'``,
-  or ``'-r'``.  No editor window is opened, even if that is the default
-  set in the Options dialog.
+The following options are available:
+
+.. option:: -c <command>
+
+   Run the specified Python command in the shell window. For example, pass ``-c "print('Hello, World!')"``.
+   On Windows, the outer quotes must be double quotes as shown.
```

Comment:
@picnixz I want to be able to make comments more than 1 line long, such as here, without having super long lines in the .rst file.  Quick reference [directives](https://devguide.python.org/documentation/markup/#directives) just says "The directive content follows after a blank line and is indented relative to the directive start."  Unindenting the line is an error and adding `\` to the previous line does nothing.  All indented lines are supposed to be part of the directive, but the above seems to handle "On Windows ..." otherwise.  It is printed unindented after a blank line.

Reply:
A newline should work? Sphinx/DocUtils only creates a new paragraph on a blank line (\n\n).
```suggestion
   Run the specified Python command in the shell window.
   For example, pass ``-c "print('Hello, World!')"``.
   On Windows, the outer quotes must be double quotes as shown.
```

Reply:
I asked because the simple newline I had did not work.  But two days later, it does.  But I like the 2nd line split and merged it.